javascript/packages/orchestrator/src/chainSpec.ts

```diff
@@ -74,9 +74,13 @@ export function clearAuthorities(specPath: string) {
     if (runtimeConfig?.collatorSelection)
       runtimeConfig.collatorSelection.invulnerables = [];
 
-    // Clear staking
+    // clear staking
     if (runtimeConfig?.staking) {
-      stakingBond = BigInt(runtimeConfig.staking.stakers[0][2]);
+      stakingBond = BigInt(0);
+      for (const staker in runtimeConfig.staking.stakers) {
+        stakingBond += BigInt(runtimeConfig.staking.stakers[staker][2]);
+      }
+
```

Comment:
```suggestion
      if ( runtimeConfig.staking.stakers[0] ) stakingBond = BigInt(runtimeConfig.staking.stakers[0][2]);
```
Hi @gpestana, in the current code we only use the value of the first staker for adding the nodes later (without add them). Something like this should work and we can change the first if condition to make the code more clean.
Thx!